sirepo/package_data/static/js/hellweg.js

```diff
@@ -153,10 +153,22 @@ SIREPO.app.controller('HellwegLatticeController', function (appState, panelState
         return '';
     };
 
-    self.itemValues = function(item) {
-        return appState.viewInfo(item.type).advanced.map(function(f) {
-            return item[f];
-        }).join(' ');
+
+    self.itemLabels = function(key) {
+        return {
+            'aperture': 'Aperture',
+            'attenuation': 'Attenuation',
```

Comment:
You can omit quotes around the keys (`{aperture: 'Aperture', ...}`) but I don't know if we have an official preference...